test/k8sT/manifests/internal-artii.crt

```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I think it's important to document wow were these certs created.

Reply:
@aanm Should this be documented in the commit message? I finally found the key to the root key in a commit msg by Jarno :D

Reply:
Yeah commit message would be good IMO